config/main.py

```diff
@@ -1946,8 +1946,11 @@ def portchannel(db, ctx, namespace):
 @click.argument('portchannel_name', metavar='<portchannel_name>', required=True)
 @click.option('--min-links', default=1, type=click.IntRange(1,1024))
 @click.option('--fallback', default='false')
+@click.option('--fast-rate', default='false',
+              type=click.Choice(['true', 'false'],
```

Comment:
@globaltrouble IMHO, better to use flags or `click.BOOL`

Reply:
In the current implementation, the syntax is the same for `fallback` and for `fast-rate`. Booth requires an argument. If change type to bool or flags then I think it should be done for both options together is separate PR